frontend/public/components/cluster-settings/basicauth-idp-form.tsx

```diff
@@ -8,6 +8,7 @@ import { ButtonBar, PromiseComponent, history, AsyncComponent } from '../utils';
 import { addIDP, getOAuthResource, redirectToOAuthPage } from './';
 import { IDPNameInput } from './idp-name-input';
 import { IDPCAFileInput } from './idp-cafile-input';
+import { ActionGroup, Button } from '@patternfly/react-core';
```

Comment:
Here and the other files:  we put external dependency imports in the first group (move this import to line 4).